project_task_code/views/project_view.xml

```diff
@@ -8,7 +8,7 @@
             <field name="arch" type="xml">
                 <field name="name" position="before">
                     <field name="code" class="oe_inline" />
-                    <label string=" - " class="oe_inline" />
+                    <label for=" - " class="oe_inline" />
```

Comment:
I was going off of this document, which said to change XML attributes called "string": https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0 I can change it back, though.